src/app/app.css

```diff
@@ -264,6 +264,10 @@ html, body, #root {
   max-width: 100%;
 }
 
+:where(.pf-theme-dark) .automated-analysis-grid-item .pf-c-label {
+  --pf-c-label--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
```

Comment:
Seems like there is a higher specificity rule somewhere:

![Screenshot from 2023-03-29 01-18-39](https://user-images.githubusercontent.com/68053619/228433859-f77e3250-373e-4875-af1b-41b6376b6b56.png)


We can add `!important` here